scripts/install.sh

```diff
@@ -66,11 +66,6 @@ cp socorro_revision.txt $BUILD_DIR/application/socorro
 cp breakpad_revision.txt $BUILD_DIR/application/socorro
 
 
-if [ "$BUILD_TYPE" == "tar" ]; then
-    pushd $BUILD_DIR/application/scripts/config
-    echo "py config files no longer exist, ignoring"
-    #for file in *.py.dist; do cp $file `basename $file .dist`; done
-    popd
```

Comment:
All this did was pushd into a directory and then popd out of it. That's essentially a no-op, so I removed this bit and inverted the `if` block.